src/System.Net.Http/src/System/Net/Http/Headers/CacheControlHeaderValue.cs

```diff
@@ -232,14 +232,14 @@ public override string ToString()
             {
                 AppendValueWithSeparatorIfRequired(sb, maxAgeString);
                 sb.Append('=');
-                sb.Append(((int)_maxAge.Value.TotalSeconds).ToString(NumberFormatInfo.InvariantInfo));
+                sb.Append((int)_maxAge.Value.TotalSeconds);
```

Comment:
Are all of these values guaranteed to be non-negative?
Otherwise `NumberFormatInfo.NegativeSign` could affect the result.

Reply:
In practice, they should be non-negative. But you're right that not all appear to be validated as such. I'll re-review and revert any that are potentially problematic.